### PR TITLE
Test/ldg 490 app write tests

### PR DESCRIPTION
**Pull request by Z4karia**

### CODE REVIEW

File: bitcoin_client/ledger_bitcoin/client.py [hunk 1]
```diff
@@ -308,6 +308,24 @@ def sign_withdraw(self, data: AcreWithdrawalData, bip32_path: str) -> str:
...
+            message_bytes = message
+
+        chunks = [message_bytes[64 * i: 64 * i + 64] for i in range((len(message_bytes) + 63) // 64)]
+
+        client_intepreter = ClientCommandInterpreter()
+        client_intepreter.add_known_list(chunks)
+
+        sw, response = self._make_request(self.builder.sign_erc4361_message(message_bytes, bip32_path), client_intepreter)
+
+        if sw != 0x9000:
```
Comment (n4l5u0r) [inline bitcoin_client/ledger_bitcoin/client.py]: Define value as constant if possible

File: src/handler/sign_erc4361_message.c [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+        // Iterate on the rest of the line
+        for (size_t j = total_bytes_read - line_start; j < parsing_buffer_len; j++) {
+            total_bytes_read++;
+            PRINTF("Total bytes read: %llu\n", total_bytes_read);
+
+            if (parsing_buffer[j] == '\n') {
+                // Save leftover for next chunk
+                leftover_len = parsing_buffer_len - (total_bytes_read - line_start);
+                memcpy(leftover, parsing_buffer + parsing_buffer_len - leftover_len, leftover_len);
+                // TODO: Handle the case where the line is too long
```
Comment (n4l5u0r) [inline src/handler/sign_erc4361_message.c]: Still TODO?

**Review by n4l5u0r — CHANGES_REQUESTED** [review]
Can you check my few comments

File: bitcoin_client/ledger_bitcoin/client.py (hunk 1 above)
Comment (Z4karia) [reply bitcoin_client/ledger_bitcoin/client.py]: Done

File: src/handler/sign_erc4361_message.c (hunk 2 above)
Comment (Z4karia) [reply src/handler/sign_erc4361_message.c]: Was already done, now we have a hard limit of 128 characters max for a given line.

I removed the TODO comment.

**Review by n4l5u0r — APPROVED** [review]

**Pull request merged**